cmd/sourced/dir/dir.go

```diff
@@ -12,8 +12,12 @@ import (
 	"github.com/pkg/errors"
 )
 
-// Path returns the absolute path for $HOME/.sourced
+// Path returns the absolute path for $SOURCED_DIR, or $HOME/.sourced if unset
```

Comment:
I'd document as:
```suggestion
// Path returns the absolute path sourced configuration directory.
// It will be defined by the SOURCED_DIR env var, or $HOME/.sourced if not defined
```